newton/examples/mpm/example_mpm_anymal.py

```
@@ -329,7 +290,25 @@ def step(self):
         self.sim_time += self.frame_dt
 
     def test(self):
-        pass
+        newton.examples.test_body_state(
+            self.model,
+            self.state_0,
+            "all bodies are above the ground",
+            lambda q, qd: q[2] > 0.2,
```

Ruff (0.13.1)

Unused lambda argument: `qd`

(ARG005)